app/styles/crate.scss

```diff
@@ -313,6 +313,13 @@
         pre {
             overflow-x: auto;
         }
+
+        p {
+            code {
+                background-color: #fff;
```

Comment:
What's the purpose of this? It's not being rendered, and if it were the text would be unreadable.